velox/exec/PrefixSort.cpp

```diff
@@ -235,6 +236,17 @@ uint32_t PrefixSort::maxRequiredBytes(
   return prefixSort.maxRequiredBytes();
 }
 
+// static
+void PrefixSort::timSort(
+    std::vector<char*, memory::StlAllocator<char*>>& rows,
+    RowContainer* rowContainer,
```

Comment:
```
const RowContainer* rowContainer
```

Reply:
The RowContainer function `rowContainer->compareRows` is not const, so we cannot mark it as const though I don't know why compareRows is not const.

Reply:
Maybe make a try in followup to see if it breaks anything? Thanks!

Reply:
Yes, in a separate follow up PR.